Formula/opensearch-dashboards.rb

```diff
@@ -0,0 +1,80 @@
+class OpensearchDashboards < Formula
+  desc "Open source visualization dashboards for OpenSearch"
+  homepage "https://opensearch.org/docs/dashboards/index/"
+  url "https://github.com/opensearch-project/OpenSearch-Dashboards.git",
+    tag:      "1.0.0",
+    revision: "b15ad258e6960003e609a70889cda5fb39f90947"
+  license "Apache-2.0"
+
+  depends_on "python@3.9" => :build
+  depends_on "yarn" => :build
+  depends_on "node@10" # Switch to `node` after https://github.com/opensearch-project/OpenSearch-Dashboards/issues/406
+
+  def install
+    inreplace "package.json", /"node": "10\.\d+\.\d+"/, %Q("node": "#{Formula["node@10"].version}")
+
+    system "yarn", "osd", "bootstrap"
+    system "node", "scripts/build", "--release", "--skip-os-packages", "--skip-archives"
+
+    cd "build/opensearch-dashboards-#{version}-darwin-x64" do
+      inreplace Dir["bin/*"],
+                "NODE=\"${DIR}/node/bin/node\"",
+                "NODE=\"#{Formula["node@10"].opt_bin/"node"}\""
+
+      inreplace "config/opensearch_dashboards.yml",
+                /#\s*pid\.file: .+$/,
+                "pid.file: #{var}/run/opensearchDashboards.pid"
+
+      (etc/"opensearch-dashboards").install Dir["config/*"]
+      rm_rf Dir["{config,data,node,plugins}"]
+
+      prefix.install Dir["*"]
+    end
+  end
+
+  def post_install
+    (var/"log/opensearch-dashboards").mkpath
+
+    (var/"lib/opensearch-dashboards").mkpath
+    ln_s var/"lib/opensearch-dashboards", prefix/"data" unless (prefix/"data").exist?
+
+    (var/"opensearch-dashboards/plugins").mkpath
+    ln_s var/"opensearch-dashboards/plugins", prefix/"plugins" unless (prefix/"plugins").exist?
+
+    ln_s etc/"opensearch-dashboards", prefix/"config" unless (prefix/"config").exist?
+  end
+
+  def caveats
+    <<~EOS
+      Data:    #{var}/lib/opensearch-dashboards/
+      Logs:    #{var}/log/opensearch-dashboards/opensearch-dashboards.log
```

Comment:
Do we also want to rename this as we do with the `opensearch` formula? Or do the reasons we did this there not apply here?

Reply:
I don't think `opensearch-dashboards` has clusters, at least it's not in their config: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/1.0.0/config/opensearch_dashboards.yml